src/components/ui/TopBar.vue

```diff
@@ -77,6 +77,7 @@
                 username: user.primaryUsername.value,
               }"
               :size="40"
+              :pictureSize="100"
```

Comment:
In general, I really don't like that we are using hard units here, calculated in the markup. I prefer it to be done in css and to be a percentage unit to be responsive.
That said, that's a bigger battle in this whole codebase.

Reply:
Yes, I agree. That to be discussed with UX. However this value is only used to get the actual pictures size (Which should be a `srcset` at some point).